src/DeviceController.cpp

```diff
@@ -111,8 +125,12 @@ namespace earlyapp
         dmesgLogPrint("EA: Waiting for Wayland socket...");
 #endif
         // Wait for the Wayland.
-        if(bWaitWL)
+	if(!m_pConf->useCsicam())
```

Comment:
If you have a good reason to postpone a waitForWayland() method call after thread creations, why don't you keep one single code? "Wait for waylaid if the camera configuration is CSI" doesn't make sense to me..

Reply:
This Csi camera path integrated in clear Linux, here we are using "useCsicam" flags  and making sure that changes doesn't effect the SOS behaviour also.

Reply:
Understand that this is for making sure other configurations. What I would like to suggest here is figuring out whether this is necessary. If you tested the single WL waiting code on SOS environment and it didn't work it leaves us no choice but if it did, I see no reason to take future maintenance risks.